### PR TITLE
[documentation] AMDGPU.jl supports rocSPARSE

### PR DESCRIPTION
**Pull request by amontoison**

### CODE REVIEW

**Comment by pxl-th** [comment]
@amontoison, https://github.com/JuliaGPU/AMDGPU.jl/pull/486 intends to fix issues observed in CI.
Specifically, we didn't support specifying buffer type in ctor.

File: test/gpu/amd.jl [hunk 1]
```diff
@@ -14,6 +14,25 @@ include("gpu.jl")
     A_gpu = ROCMatrix(A_cpu)
     b_gpu = ROCVector(b_cpu)
     x, stats = minres(A_gpu, b_gpu)
+    r_gpu = b_gpu - A_gpu * x
+    @test norm(r_gpu) ≤ 1e-4
+
+    A_cpu = sprand(100, 200, 0.3)
+    b_cpu = rand(100)
+    A_csc_gpu = ROCSparseMatrixCSC(A_cpu)
```
Comment (pxl-th) [inline test/gpu/amd.jl:22]: Access to sparse matrices requires `using AMDGPU.rocSPARSE`.

File: test/gpu/amd.jl [hunk 2]
```diff
@@ -14,6 +14,25 @@ include("gpu.jl")
...
+    A_csr_gpu = ROCSparseMatrixCSR(A_cpu)
+    A_coo_gpu = ROCSparseMatrixCOO(A_cpu)
+    b_gpu = ROXVector(b_cpu)
```
Comment (amontoison) [inline test/gpu/amd.jl:25]:
```suggestion
```

**Comment by amontoison** [comment]
@pxl-th
I opened this [PR](https://github.com/JuliaGPU/AMDGPU.jl/pull/488) to fix the issue with ROCSparseMatrixCOO.

**Pull request merged**